app/web/components/HorizontalScroller.tsx

```diff
@@ -56,13 +56,13 @@ export default function HorizontalScroller({
     >
       {children}
       {fetchNext && hasMore && (
-        <div>
+        <Box display="flex" alignItems="center">
```

Comment:
I think the main reason was to avoid splitting styles between two places. So yeah, we switched over to preferring `makeStyles` and `<div className={classes.x}>`

Reply:
Sounds good! I'll update all my code here and in future contributions to using class names